src/test/end-to-end/codecept.conf.js

```diff
@@ -1,15 +1,15 @@
 const testConfig = require('src/test/config.js');
 
 exports.config = {
-    'tests': './paths/**/*.js',
+    'tests': './paths/**/caveatPath.js',
     'output': './output',
     'helpers': {
         'Puppeteer': {
             'url': testConfig.TestE2EFrontendUrl || 'http://localhost:3000',
             'waitForTimeout': 60000,
             'getPageTimeout': 60000,
             'waitForAction': 1500,
-            'show': false,
+            'show': true,
```

Comment:
could pass a variable that is entered that can turn this on or off?
`process.env.SHOW_BROWSER_WINDOW || false`.